README.md

```diff
@@ -68,6 +68,14 @@ To **update** it to the latest version, run this instead:
 brew upgrade accio
 ```
 
+### Via [Mint](https://github.com/yonaskolb/Mint):
+
+To **install** the latest version of Accio, run this command:
```

Comment:
Would you mind adjusting this line so it gets clear the `install` command also updates Accio if it's already installed? That would be more in-line with the Homebrew installation instructions 😉

Maybe change it to `To **install** Accio or **update** to the latest version of Accio, run this command:`

Reply:
Done :)